Cubical/Data/Int/Divisibility.agda

```diff
@@ -21,6 +21,7 @@ open import Cubical.Data.Nat.Divisibility
 open import Cubical.Data.Nat.Mod
 open import Cubical.Data.Int
   hiding   (_+_ ; _·_ ; _-_ ; -_ ; addEq ; ·Comm ; ·Assoc ; +Comm ; +Assoc)
+  renaming (·DistL+ to ·ℤDistL+)
```

Comment:
Is this really being used? Otherwise it might be better to hide

Reply:
now hidden